arm/comm.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 const errCodeCollision = 0x1F
+const servoMotionMode = 1
```

Comment:
Would be nice to have other modes documented here as well to know what else is available

Reply:
I think the map in `constants.go` could use a little love, but not for her pr.

Reply:
It looks like there's only one other mode we ever set

Reply:
Yes - you're right - but there are other modes, and we should theoretically have all of the modes and error codes as enums  in a Go file with proper documentation, but it's cleanup work that I think should be done in one go.